src/Project.Library/Application.cpp

```diff
@@ -30,8 +30,14 @@ void Application::Run()
 
     spdlog::info("App: Loaded");
 
+    deltaTime = 0.0;
+    double prevFrame = glfwGetTime();
```

Comment:
looks alright besides wording...

`prevFrame` should perhaps be `previousTime`
and `curFrame` should be `currentTime` we are also not deltaFrame-ing but deltaTime-ing :)

deltaTime should perhaps also be a float, because most of the time we use floats to calculate anything and would need to cast that double.

Also you could pass `deltaTime` along to Update() and Render() and dont need `Application::GetDeltaTime()` However if it stays with that getter, `deltaTime` should then be `_deltaTime` as its a private member field and i believe other private members are named `_camelCase`.

What do you think?

Reply:
I'll make the changes. Good suggestions 👍

Reply:
I thought about both the passing deltaTime vs the Getting deltaTime thing. Both seem good but I think passing deltaTime may be a good way to discourage 'global dependency' since deltaTime can be used in Update() functions inside objects and other systems in someone's project, and passing it as a parameter would allow those systems to use a deltaTime without having to consider 'how do I retrieve it from an Application class'